.github/workflows/conformance.yaml

```diff
@@ -24,7 +24,5 @@ jobs:
           distribution: 'temurin'
           java-version: '17'
           cache: 'gradle'
-      - name: Generate
-        run: make checkgenerate
```

Comment:
I had to add this step in the previous PR because it would fail unless the generation step is done before. This would also break the release flow. By moving the logic and dependencies to the Gradle build, we can ensure that prior to compiling Java code we run necessary code generation steps first.